packaging/torcharrow/meta.yaml

```diff
@@ -11,49 +11,39 @@ requirements:
     # - { { compiler('cxx') } } # [win]
     - cmake
     - python
-    {% if osx %}
-    - boost
-    {% elif linux %}
-    - ninja-build
-    - checkinstall
-    - libssl-dev
-    - libboost-all-dev
-    - libdouble-conversion-dev
-    - libgoogle-glog-dev
-    - libgflags-dev
-    - libevent-dev
-    - libre2-dev
-    {% endif %}
   host:
     - python
     - setuptools
     {{ environ.get('CONDA_PYTORCH_BUILD_CONSTRAINT') }}
   run:
     - python
     - numpy
-    - boost
-    - libboost
     - typing_inspect
     - pandas
     - typing
     - tabulate
     - pyarrow
     - arrow
     - cffi
+    - glog 0.4.0
     {{ environ.get('CONDA_PYTORCH_CONSTRAINT') }}
 
 build:
   string: py{{py}}
   script_env:
     - BUILD_VERSION
 
+#GitHub Actions usually provide Mac without AVX2 support, for now just skip the test until Velox can be run without AVX2
```

Comment:
noob question: is this file read by the command like `conda build`?

Reply:
Yes. This is the file to specify all configuration of build/test/run for conda package.